### PR TITLE
Add ApiResponse tags, and sample responses to techs, users, teams, sprints, forms, auth endpoints

### LINKED ISSUE #67: Add @ApiResponse tags in controller classes already implemented (techs, users, teams, sprints, auth)
#### This is a part of maintaining API endpoints documentation.
* [ ] Auths
* [ ] Sprints
* [ ] Techs
* [ ] Teams
* [ ] Users

#### All API Creations/Changes:
* [ ] Have you added satisfactory documentation to Swagger (for example: @ApiParam, ApiOperation, @ApiCreatedResponse, @ApiOkResponse, @ApiNotFoundResponse, @ApiBadRequestResponse)?

### PR DESCRIPTION
**Pull request by cherylli**
# Description

Add ApiResponse tags, and sample responses to techs, users, teams, sprints, forms, auth endpoints
Also some minor changes to the endpoints, and database (spelling mistakes, sprintPerHr type)

## Issue link

Fixes #67 (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

_**Tests with auth failed as reason described in discord**_

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by curtwl — APPROVED** [review]
LGTM, everything working in Swagger.

File: src/global/responses/errors.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { ApiProperty } from "@nestjs/swagger";
+
+export class ConflictErrorResponse {
```
Comment (smurph7894) [inline src/global/responses/errors.ts:3]: Is this a universal error for all modules? The example message may not be generic enough if so. Alternatively maybe add nestJS and team to the class name so it doesn't get used for like a querying conflict or module other than teams.

**Review by jenny-alexander — APPROVED** [review]
Reviewed branch in Swagger - LGMT.

**Review by curtwl — APPROVED** [review]
LGTM

**Pull request merged**